### DISCUSSION TITLE
Neu Devices

### DISCUSSION THREAD

**Discussion by mahrens61** [thkl/hap-homematic#663] [General]
Wie bekomme ich den Rauchwarner HmIP-SWSD und den Wasserwächter HmIP-SWD ins Homekit?

**Comment by schoe-to** [comment]
Konnte ich einfach hinzufügen, wird dann im jeweiligen Raum angezeigt...

![image](https://github.com/thkl/hap-homematic/assets/151659531/766fd438-49e8-4ff4-9e48-a94cc0d08b74)


![image](https://github.com/thkl/hap-homematic/assets/151659531/02637842-bcd1-43d2-a606-d4aa9daa26a8)

**Comment by mahrens61** [comment]
Danke für das Feedback. Jetzt funktioniert es auch bei mir.
Es war in der schrägen HomeKit Oberfläche schwer zu finden.


Viele Grüße
Best regards
Avec mes meilleures salutations 
Cordiali saluti

Dr. h.c. Dipl.-Ing.(FH) Michael Ahrens

Meisenweg 25
CH-3604 Thun

Mobile: +41 76 460 35 81
